spec/classes/foreman_config_spec.rb

```diff
@@ -161,6 +162,17 @@ class {'foreman':
         end
       end
 
+      describe 'with unattended_url' do
+        let :pre_condition do
+          "class {'foreman':
+            unattended_url => 'http://example.com',
+          }"
+        end
+
+        it { should contain_concat__fragment('foreman_settings+01-header.yaml').
+            with_content(/^:unattended_url:\s*http:\/\/example.com$/) }
```

Comment:
You can use ```with_content(%r{^:unattended_url:\s*http://example.com$})```

Reply:
@ekohl 
  Thanks, I couldn't remember the regex pattern syntax and forgot to go back to it :)